### DISCUSSION TITLE
Full RTL Support

### DISCUSSION THREAD

**Discussion by aliwesome** [laravel/nova-issues#3858] [Ideas & Feature Requests]
We need to use nova with RTL support. There is a [nice package](https://github.com/pishran/nova-rtl-theme)[](https://user-images.githubusercontent.com/8510486/74316310-fd0f9c00-4d81-11ea-83c5-21d12a7ebf14.png) that helps to migrate nova's design to support RTL.

However, there are a few issues with Dropdown alignments. Some components' placement is "fixed" in the source code. For example, the views/Index.vue component declares a SelectAll dropdown (around line 87). The initiation of the dropdown states that the placement should be bottom-end, which makes the dropdown open from the left edge in RTL context.

Ideally, the placement of the dropdown should be automatically calculated instead of being hard-coded. [The docs](https://www.npmjs.com/package/v-tooltip) for v-tooltip states that one of the possible placements is bottom which,I guess , will automatically calculate the left/right position.

Alternatively, it would be great if nova had an isRtl config (probably declared in the config/nova.php file which will cascade down to the frontend (and potentially will be embedded in every component through a mixin/vuex). Then in the components themselves, we can dynamically figure out what the alignment for the dropdowns should be.

**Comment by crynobone** [comment]
Ideally, we would look into RTL only after upgrading to Tailwind 3 and avoid depending on 3rd party solution. There is some work on adding Tailwind 3 but it's still very much a **WIP**™

**Comment by bdsoha** [comment]
Any updates on this topic?

**Reply by crynobone** [reply]
https://nova.laravel.com/docs/installation.html#enabling-rtl-support
